pages/common/jj.md

```diff
@@ -6,16 +6,24 @@
 
 - Update description of the revisions specified by given revsets (e.g. `B::D`, `A..D`, `B|C|D`, etc.):
 
-`jj {{[desc|describe]}} {{[-r|--revision]}} {{revsets}}`
+`jj describe {{[-m|--message]}} "{{message}}" {{[-r|--revision]}} {{revsets}}`
```

Comment:
```suggestion
`jj {{[desc|describe]}} {{[-m|--message]}} "{{message}}" {{[-r|--revision]}} {{revsets}}`
```
This is still missing shortform of describe

Reply:
I've added back both short and long here.

This feels counter to the style guide, which says to always use the long option for args. And it seems less readable, given the otherwise concise output of most commands that I read in `tldr`. But if that's the direction you want to take this project, I'm happy to modify.

A feature and/or oddity of `jj` is that you can partially shorten any flag or subcommand as long as it's uniquely identifiable. So "desc, descr, descri, describ, describe." It would be too verbose to enumerate all those, so it seems best to stop at the long version and let people figure out shortcuts and optimizations for themselves.

But again, happy to include both if that's where you want to take this project.

Reply:
That is not what the style guide says. It instructs to use long options over short options when they are available but using option placeholders is fine as well.
The option placeholders here are interpreted on the client side so that the client only shows long version. If you run up-to-date software, this won't show up for you unless you call for it.